bin/mpiexecjl

```diff
@@ -61,7 +61,10 @@ fi
 SCRIPT='
 using MPI
 ENV["JULIA_PROJECT"] = dirname(Base.active_project())
-mpiexec(exe -> run(`$exe $ARGS`))
+mpiexec()
+p = mpiexec(exe -> run(pipeline(`$(exe) $(ARGS)`; stdout, stderr); wait=false))
```

Comment:
Is the `wait=false` needed?

Reply:
I don't know how to get the process object and access the exit status otherwise, but maybe there are better ways to do this.

Reply:
`run` should return the process object independent of if you pass `wait` or not?

Reply:
Maybe I'm missing something obvious, but if the process exits with non-zero status you don't have anything:
```console
% julia -e 'proc = run(`false`); @show proc'
ERROR: failed process: Process(`false`, ProcessExited(1)) [1]

Stacktrace:
 [1] pipeline_error
   @ ./process.jl:598 [inlined]
 [2] run(::Cmd; wait::Bool)
   @ Base ./process.jl:513
 [3] run(::Cmd)
   @ Base ./process.jl:510
 [4] top-level scope
   @ none:1
 [5] eval
   @ ./boot.jl:432 [inlined]
 [6] exec_options(opts::Base.JLOptions)
   @ Base ./client.jl:291
 [7] _start()
   @ Base ./client.jl:533
```

Reply:
> If wait is false, the process runs asynchronously. You can later wait for it and
  check its exit status by calling success on the returned process object.

ah of course...